### PR TITLE
Addition of prev example to build

### PR DESCRIPTION
**Pull request by sshanks-kx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,3 +26,16 @@ If you wish to deploy the library to your machines local repository, in order to
 `mvn clean install`
 
 Please refer to the [Apache Maven documentation](https://maven.apache.org/guides/index.html) for further details
+
+## Code Example
+
+The following describes each with an example of how to run from Maven (note: Maven is not required to run the applications, but used here for convenience.
```
Comment (awilson-kx) [inline README.md:32]: same again -missing close bracket

File: README.md [hunk 2]
```diff
@@ -26,3 +26,16 @@ If you wish to deploy the library to your machines local repository, in order to
 `mvn clean install`
 
 Please refer to the [Apache Maven documentation](https://maven.apache.org/guides/index.html) for further details
+
+## Code Example
+
+The following describes each with an example of how to run from Maven (note: Maven is not required to run the applications, but used here for convenience. 
+
+The example remotely creates a table 't', then queries KDB+ for its contents, which it then displays its columns/data to the console
```
Comment (awilson-kx) [inline README.md:34]: usually we put kdb+ in lower case

File: README.md [hunk 3]
```diff
@@ -26,3 +26,16 @@ If you wish to deploy the library to your machines local repository, in order to
...
+
+`mvn clean install` should be performed prior to running.
+
+The jdbc driver passes the q or sql text to the server. For SQL support you should take the ps.k file from the odbc3 zip file [here](https://code.kx.com/v2/interfaces/q-server-for-odbc3/)
+and ensure that is loaded into your kdb+ process. ps.k is the sql transpiler or enquire about SQL support with KX Insights. The example requires KDB+ to be listening on TCP port 5001
```
Comment (awilson-kx) [inline README.md:39]: usually we put kdb+ in lower case

**Review by awilson-kx — COMMENTED** [review]
looks good, couple of little things

**Comment by sshanks-kx** [comment]
changes done

**Review by awilson-kx — APPROVED** [review]
looks good

**Pull request merged**